src/js/app.js

```
@@ -4,8 +4,9 @@
 // ══════════════════════════════════════════════
 // GLOBAL STATE & COMPATIBILITY LAYER
 // ══════════════════════════════════════════════
-let filteredOrgs = [];
-let MENTOR_DATA = {};
+// filteredOrgs declared in index.html inline script
+/* global filteredOrgs, visibleCount */
```

ESLint

[error] 'visibleCount' is already defined by a variable declaration.

(no-redeclare)